src/libslic3r/GCode.cpp

```diff
@@ -5341,6 +5341,10 @@ std::string GCode::_extrude(const ExtrusionPath &path, std::string description,
             if (m_config.overhang_speed_classic.value && m_config.enable_overhang_speed.value) {
```

Comment:
This doesn't look right to me, because `overhang_speed_classic` should have been disabled and will never be true.